test/e2e/secret_test.go

```diff
@@ -2,6 +2,7 @@ package integration
 
 import (
 	"fmt"
+	"io/ioutil"
```

Comment:
sorry I was just looking at https://github.com/containers/podman/pull/15893 before
You have to use io directly